src/patterns/components/inputs/textarea-state-switcher.hbs

```diff
@@ -1,8 +1,8 @@
 ---
-name: Search Input State Switcher
+name: Textarea State Switcher
```

Comment:
Fixed the name of this, it was labeled as Search input when its actually the textarea.